rest/rest-api/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SchedulerRestClient.java

```diff
@@ -95,6 +119,20 @@ public SchedulerRestClient(String restEndpointURL, ClientHttpEngine httpEngine)
         scheduler = createRestProxy(providerFactory, restEndpointURL, httpEngine);
     }
 
+    private void setBlindTrustSSLContext() {
+        try {
+            TrustStrategy acceptingTrustStrategy = (cert, authType) -> true;
+            sslContext = SSLContexts.custom().loadTrustMaterial(null, acceptingTrustStrategy).build();
+
+        } catch (KeyStoreException | KeyManagementException | NoSuchAlgorithmException e) {
+            e.printStackTrace();
```

Comment:
What about throwing a runtime exception instead of printStackTrace ?

Reply:
This would imply to change the calls like:

org.ow2.proactive_grid_cloud_portal.cli.cmd.sched.FreezeCommand

```java
   @Override
    public void execute(ApplicationContext currentContext) throws CLIException {
        SchedulerRestInterface scheduler = currentContext.getRestClient().getScheduler();
        try {
       } catch (Exception e) {
            handleError("Error occurred while trying to freeze the scheduler:", e, currentContext);
        }
    }
```
to
```java
   @Override
    public void execute(ApplicationContext currentContext) throws CLIException {
   
        try {
      SchedulerRestInterface scheduler = currentContext.getRestClient().getScheduler();

       } catch (Exception e) {
            handleError("Error occurred while trying to freeze the scheduler:", e, currentContext);
        }
    }
```

Reply:
the problem is that e.printStackTrace() will most likely go to /dev/null :-)

Reply:
you need to change calls to currentContext.getRestClient() according to my comment to avoid uncatched exceptions

Reply:
Done.